lib/ash/error/error.ex

```diff
@@ -11,6 +11,23 @@ defmodule Ash.Error do
     ],
     unknown_error: Ash.Error.Unknown.UnknownError
 
+  @type ash_errors ::
+          Ash.Error.Forbidden.t()
+          | Ash.Error.Invalid.t()
+          | Ash.Error.Framework.t()
+          | Ash.Error.Unknown.t()
+          | Ash.Error.Unknown.UnknownError.t()
```

Comment:
`ash_errors` here includes any of a huge list of errors, not just the classes. It also includes custom errors defined by end users that use one of our error classes. I think the best we can do here is `Exception.t()`.